### DISCUSSION TITLE
Projects with low Build Activity - Survey

### DISCUSSION THREAD

**Discussion by jadsonjs** [infection/infection#1627] [General]
Dear Developers of project **infection/infection**,

We are conducting a study on CI metrics and we noticed that in your project, something peculiar happened.

During the period you are using Travis CI, the project passed by a long period of inactivity.
We noticed that the TravisCI configuration file (.travis.yml) was committed to your project on the date of **2017-07-01**.
But there is only a build history in TravisCI for a very short period, between **2020-02-28** and **2020-03-26**, in which were performed suddenly **516** builds.

Please, we would like you to answer this short survey with just 4 questions to help us identify the cause of this behavior.

https://forms.gle/UkkFrxpi7MNscYmV7

**Comment by sanmai** [comment]
IIRC, TravisCI were migrating everyone to their .com offering at that time, only later to cut it off virtually everyone.
